kinbot/constants.py

```diff
@@ -32,31 +37,55 @@
 st_bond['NO'] = 1.49*1.2
 st_bond['NS'] = 1.6*1.2
 st_bond['NN'] = 1.35*1.2
+st_bond['FF'] = 1.55*1.2
+st_bond['FO'] = 1.41*1.2
+st_bond['FH'] = 0.91*1.2
+st_bond['ClCl'] = 1.99*1.2
+st_bond['ClO'] = 1.70*1.2
+st_bond['ClH'] = 1.275*1.2
+st_bond['BrBr'] = 2.29*1.2
+st_bond['II'] = 1.48*1.2
 
 st_bond['ts'] = 1.15
 st_bond['S'] = 6  # maximum valence, can also be 2 or 4
 st_bond['C'] = 4
 st_bond['O'] = 2
 st_bond['H'] = 1
 st_bond['N'] = 5  # maximum valence, can also be 3
+st_bond['F'] = 1
+st_bond['Cl'] = 1
+st_bond['Br'] = 1
+st_bond['I'] = 1
 
 mass = {'H': 1}
 mass['C'] = 12
 mass['N'] = 14
 mass['O'] = 16
 mass['S'] = 32
+mass['F'] = 19
+mass['Cl'] = 35  # isotopes = Cl-35, Cl-37, Cl-36 (3 most stable/common)
+mass['Br'] = 80  # isotopes = Br-79, Br-81, Br-77 (3 most stable/common)
```

Comment:
If there is no 80 in the isotopes, why 80?

Reply:
I was going off mass on the periodic table, which is 79.9, so it got rounded to 80. Br-79 has an abundance of about 50.7%, while Br-81 has an abundance of about 49.3%

So i can change it to either one.

Reply:
ok, fine, it's for the chemid only I think, there is the exact mass constant that really matters for frequencies. We need to think of isotopes at some stage, but but no now.